pages_builder/pages/search-result.yml

```diff
@@ -0,0 +1,96 @@
+pageTitle: Notification banners - Test toolkit
+assetPath: govuk_template/assets/
+head: >
+  <link rel="stylesheet" media="all" type="text/css" href="public/stylesheets/index.css" />
+  <link rel="stylesheet" media="all" type="text/css" href="public/stylesheets/grids/index.css" />
+  <link rel="stylesheet" media="all" type="text/css" href="public/stylesheets/_search-result-wrapper.css" />
+content: >
+  <div id="global-breadcrumb" class="header-context">
+    <nav>
+      <ol class="group" role="breadcrumbs">
+        <li>
+          <a href="./">Home</a>
+        </li>
+      </ol>
+    </nav>
+  </div>
+  <main role="main" id="content" class="wrapper">
+    <div id="wrapper">
+      <header class="page-heading">
+        <h1>Search result</h1>
+      </header>
+      <div class="grid-row">
+        <div class="two-third-column">
+          <div class="search-result">
+            <h1 class="search-result-title">
```

Comment:
These search results headings should be h2s I think as we already have a h1 for this page.